### PR TITLE
First read edits

### PR DESCRIPTION
**Pull request by djoyner**
This fixes a small set of things I either stumbled over or just noticed during a first read. Separated into 3 commits in case you disagree w/any edits and want to `cherry-pick`. I can squash into one commit if you'd prefer that.

There were a few other things I stumbled over. One was a possible intuition for functions to go along with "operators are like functions in programming languages". When functions were first introduced, I didn't completely grok their definition and went back to _Specifying Systems_ to try to understand these better. From that, it seems they are like mathematical functions with finite domains and ranges. For me this would have been a helpful metaphor at the time, but I wasn't 100% confident in making the edits. But it did seem like you mentioned something very similar later on.

As for the rest, I think it's all me. The last few sections introduce a whole lot of power. ;)

Thank you so much for writing this. I can appreciate that it must have taken a lot of effort. I'm looking forward to your book.

### CODE REVIEW

File: content/introduction/example/_index.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Can you transfer a negative amount of money? We could add an `assert money > 0`
...
 
-How is this different from `assert`? Assert checks in one place. We can specify MoneyNotNegative as an _Invariant_ of the system, something that must be true in all possible system states. It'll check before money is pulled from Alice's account, being the deposit and the withdrawal, etc. If we added a `money := money - 2` step anywhere the MoneyNotNegative invariant will catch that the spec fails when `money = 1`.
+How is this different from `assert`? Assert checks in one place. We can specify MoneyNotNegative as an _Invariant_ of the system, something that must be true in all possible system states. It becomes part of the model, and then it'll check before money is pulled from Alice's account, being the deposit and the withdrawal, etc. If we added a `money := money - 2` step anywhere the MoneyNotNegative invariant will catch that the spec fails when `money = 1`.
```
Comment (hwayne) [inline content/introduction/example/_index.md]: What do you think about "we can tell TLA+ to include it in the model, and then..."

File: content/introduction/example/_index.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ PlusCal supports multiprocess algorithms. The processes can be completely differ
 
 The accounts are global variables, while money is a local variable to the process. This means that there are 400 possible initial states, as the first transfer can be one dollar and the second seven. However, there are actually 2400 possible behaviors! That's because TLC can choose which order to run the processes in, as well as how to interleave them.
 
-However, `MoneyNotNegative` doesn't make sense anymore, since there's now two values for money. Make sure to uncheck that invariant to avoid an error and rerun. You should get the following error:
+However, `MoneyNotNegative` doesn't make sense anymore, since there's now two values for money. If you'd added that to the model make sure to uncheck it to avoid an error and rerun. You should get the following error:
```
Comment (hwayne) [inline content/introduction/example/_index.md]: ✔

File: content/pluscal/toolbox/_index.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Terms:
...
 
-We can run the model by clicking the green arrow in the top left. TLC will explore the entire state space, looking for possible failures in our invariants. Since we don't actually have any invariants yet, it won't find any failures, so this will be successful. Notice that it lists of the diameter as 2 and the number of distinct states as 2. The former means that the longest interesting progression was two steps long: the initial state where `x = 5` and the `Add` step where `x = 6`. Since there's only one possible starting state, there are 2 distinct states total.
+We can run the model by clicking the green arrow in the top left. TLC will explore the entire state space, looking for possible failures in our invariants. Since we don't actually have any invariants yet, it won't find any failures, so this will be successful. Notice that it lists the diameter as 2 and the number of distinct states as 2. The former means that the longest interesting progression was two steps long: the initial state where `x = 5` and the `Add` step where `x = 6`. Since there's only one possible starting state, there are 2 distinct states total.
```
Comment (hwayne) [inline content/pluscal/toolbox/_index.md]: Think this is a false positive

File: content/tla/operators.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title = "Operators"
 weight = 1
 +++
 
-An operator is a thing that does a thing. It's what programmers might refer to as a function. You use a `==` for them.
+An operator is a thing that does a thing. It's what programmers might refer to as a function. You use a `==` to define them.
```
Comment (hwayne) [inline content/tla/operators.md]: "You define them with `==`."

File: content/introduction/example/_index.md (hunk 1 above)
Comment (djoyner) [reply content/introduction/example/_index.md]: Fine by me.

File: content/pluscal/toolbox/_index.md (hunk 3 above)
Comment (djoyner) [reply content/pluscal/toolbox/_index.md]: It's all but invisible in the diff: the word `of` was deleted from "Notice that it lists of the diameter as 2...".

File: content/tla/operators.md (hunk 4 above)
Comment (djoyner) [reply content/tla/operators.md]: Definitely better.

**Pull request merged**